.github/release.yml

```diff
@@ -0,0 +1,14 @@
+changelog:
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking-change
+    - title: New Features 🎉
+      labels:
+        - enhancement
+    - title: Bug fixes
+      labels:
+        - bug
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
you can mark pull requests (even if they are already closed) with proper labels and try creating new release and then click `generate release notes` to see how it would look like

Reply:
Even though breaking changes may have never happened so far (as far as I'm aware), I think this is a good set to start with.